<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HyperMediaEditorClass</class>
 <widget class="QMainWindow" name="HyperMediaEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1185</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1185</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HyperMediaEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="originBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Origin Video</string>
    </property>
    <widget class="QSlider" name="leftSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>541</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Frame" name="leftWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>528</width>
       <height>432</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>21</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Origin Video Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="originPathLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="originPathButton">
        <property name="text">
         <string>Choose Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="originLoadButton">
        <property name="text">
         <string>Load Video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="originSetStartTimeButton">
        <property name="text">
         <string>Set Start Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="originJumpToStartButton">
        <property name="text">
         <string>Jump to Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="originSelectedStartTimeLabel">
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectArea">
        <property name="text">
         <string>Select Area</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="originSelectedEndTimeLabel">
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="originSetEndTimeButton">
        <property name="text">
         <string>Set End Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="originJumpToEndButton">
        <property name="text">
         <string>Jump to End</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="originPlayButton">
        <property name="text">
         <string>Play/Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="originStopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>248</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="originCurrentTimeLabel">
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="originVideoLengthLabel">
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="targetBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>571</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Origin Video</string>
    </property>
    <widget class="QSlider" name="rightSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>541</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Frame" name="rightWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>528</width>
       <height>432</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>21</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Target Video Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetPathLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="targetPathButton">
        <property name="text">
         <string>Choose Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="targetLoadButton">
        <property name="text">
         <string>Load Video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="targetSetTimeButton">
        <property name="text">
         <string>Set Target Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="targetJumpToTargetButton">
        <property name="text">
         <string>Jump to Target</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="targetSelectedTimeLabel">
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>541</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="targetPlayButton">
        <property name="text">
         <string>Play/Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="targetStopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>248</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="targetCurrentTimeLabel">
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetVideoLengthLabel">
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>1161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select Link</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectLinkComboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Link Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linkNameLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QPushButton" name="setLinkButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Set Link</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeLinkButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Remove Link</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveFileButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save File</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Frame</class>
   <extends>QWidget</extends>
   <header>../../Classes/frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="HyperMediaEditor.qrc"/>
 </resources>
 <connections/>
</ui>
